include/fastdds/rtps/transport/TransportInterface.h

```diff
@@ -232,6 +232,11 @@ class RTPS_DllAPI TransportInterface
     {
     }
 
+    /**
+     * @brief Update network interfaces.
+     */
+    virtual void update_network_interfaces() = 0;
```

Comment:
If we don't want to break API, this method should have a default empty implementation.
Otherwise, a user that has already created a custom transport, will be forced to change their code so it can build correctly.

Reply:
Done 45f424b79